main.tex

```diff
@@ -34,6 +34,90 @@
 \usepackage{listings-golang}
 \usepackage{adjustbox}
 \usepackage{multicol}
+\usepackage{glossaries}
+
+\makeglossaries
+
+\newglossaryentry{chair}
+{
+        name=Chair,
+        description={Elected leader of the agents}
+}
+\newglossaryentry{manifesto}
+{
+        name=Manifesto,
+        description={Conditions with which agents run for election as Chair}
+}
+\newglossaryentry{fight proposal}
+{
+        name=Fight Proposal,
+        description={A list of thresholds to determine an agents fight decision}
+}
+\newglossaryentry{loot}
+{
+        name=Loot,
+        description={Equipment dropped by defeated enemies, including swords, shields, health potions and stamina potions}
+}
+\newglossaryentry{potions}
+{
+        name=Potions,
+        description={Drank by agents to restore health points or stamina points}
+}
+\newglossaryentry{bonus attack}
+{
+        name=Bonus Attack,
+        description={Value of a sword equiped by an agent}
+}
+\newglossaryentry{bonus defense}
+{
+        name=Bonus Defense,
+        description={Value of a shield equiped by an agent}
+}
+\newglossaryentry{common pool resource}
+{
+        name=Common Pool Resource,
+        description={Loot dropped my enemy to be distributed amoungst agents}
+}
+\newglossaryentry{institutionalised power}
+{
+        name=Institutionalised Power,
+        description={the power wielded by the Chair to control agents and direct their behavior through the use of rewards and punishments}
+}
+\newglossaryentry{sanctions}
+{
+        name=Sanctions,
+        description={Methods by which the Chair punishes disobedience}
+}
+\newglossaryentry{alternative dispute resolution}
+{
+        name=Alternative Dispute Resolution,
+        description={Settling disputes outside of formal proceedings with the assistance of an impartial dispute resolution body}
+}
+\newglossaryentry{social capital}
+{
+        name=Social Capital,
+        description={Social value calculated by an agent and assigned to another}
+}
+\newglossaryentry{utility}
+{
+        name=Utility,
+        description={Value of worth to the community or self, defined by an agent and assigned to another}
+}
+\newglossaryentry{agent}
+{
+        name=Agent,
+        description={A player within the game}
+}
+\newglossaryentry{equipment}
+{
+        name=Equipment,
+        description={Loot dropped by defeated enemies that includes; Swords, Shields, Heath Potions and Stamina Potions}
+}
+\newglossaryentry{trusted social network}
+{
+        name=Trusted Social Network,
+        description={The group of trusted agents within an agents social network, where 'trust' is individually defined by each agent}
```

Comment:
Should this be in the global glossary? It might be more team-specific how we define "trust" and things like social capital. Might be confusing if some teams give it a slightly different meaning.

Reply:
sure, i've said that it is define individually. But the trusted social network is the same definition for all agents, no?

Reply:
ah true, then it's probably general enough